### PR TITLE
update contributing guidelines

### PR DESCRIPTION
**Pull request by TessFerrandez**
Updates the contributing guidelines to be shorter and more concise
Also adds a bit about what content and what tone is expected

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,91 +1,69 @@
...
+### Permissions & Contributions
 
-```plaintext
-<user alias>/<work item ID>_<title>
-```
+There are two ways in which you can help update the content:
 
-### Linting
+- **One-off:** \
+If you are not a regular contributor to the project, but you would like to contribute some changes, the best way to do it is to follow these [steps](#how-to-contribute)
```
Comment (shiranr) [inline CONTRIBUTING.md:54]: This is a little confusing as this is forwarding to the same section

**Review by shiranr — APPROVED** [review]
LGTM I left a small comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (TessFerrandez) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,91 +1,69 @@
...
+If you are unsure if your content will match this playbook, you are welcome to contact one of the [Engineering Fundamentals Champs](https://github.com/microsoft/code-with-engineering-playbook/blob/main/.github/reviewers.yml) to discuss the contribution.
 
-## What to contribute
+## General guidance
 
-Contribute to the playbook repo anything that is:
+- Quality is more important than quantity
+- Write in a way that you would want someone else to explain something to you. Use [`plain english`](http://www.plainenglish.co.uk/how-to-write-in-plain-english.html)
+- Be friendly, technical, professional and concise
+- Communicate as engineers, not marketing. We want to share what we learned, not "sell" our ideas.
```
Comment (tompaana) [inline CONTRIBUTING.md:23]:
```suggestion
- Communicate as engineers, not marketing; we want to share what we learned, not "sell" our ideas
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,91 +1,69 @@
...
-We have automatic quality gates for pull requests.
-As such, we've provided tooling to lint locally through `npm`.
+### Submitting a PR
 
-1. [Install npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
-2. Run `npm install -g npx`
-3. Run `npx mega-linter-runner --install` at the root of this repository.
-4. Run `npx mega-linter-runner` to run all linters.
+- Add your changes to a new branch `<github alias>/<title>` or by forking the repository
+- Open a PR with a clear description of the changes.
```
Comment (tompaana) [inline CONTRIBUTING.md:64]:
```suggestion
- Open a PR with a clear description of the changes
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,91 +1,69 @@
...
+- Tag the PR with the appropriate area, and link any issues that the PR closes
+- Add reviewers (you will need 2 reviewers to merge) to the PR, for example the [maintainers](#maintainers) or anyone from the [EF champs team](https://github.com/microsoft/code-with-engineering-playbook/blob/main/.github/reviewers.yml).
```
Comment (tompaana) [inline CONTRIBUTING.md:66]:
```suggestion
- Add reviewers (you will need 2 reviewers to merge) to the PR, for example the [maintainers](#maintainers) or anyone from the [EF champs team](https://github.com/microsoft/code-with-engineering-playbook/blob/main/.github/reviewers.yml)
```

**Review by tompaana — APPROVED** [review]
LGTM, few suggestions added for consistency.

**Pull request merged**